common/src/main/kotlin/com/intellij/lang/psi/LiteralTextEscaperBase.kt

```diff
@@ -8,9 +8,9 @@ package com.intellij.lang.psi
 import com.intellij.openapi.util.TextRange
 import com.intellij.psi.LiteralTextEscaper
 import com.intellij.psi.PsiLanguageInjectionHost
-import java.lang.StringBuilder
 
 /** See [com.intellij.psi.impl.source.tree.injected.StringLiteralEscaper] */
+@Suppress("KDocUnresolvedReference")
```

Comment:
It it is really not resolved, let's just remove the reference (just wrap it into backticks)